index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portforio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+ <header>
+      <img src="images/My Logo.png">
+        <img src="images/Union.svg">
+ </header>
+      <div class="container">
+         <section>
+            <div id="text1">
+            <h2>I’m Eddy Glad to see you!</h2>
+        </div>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</p>
+      
+             <h4>LET’S CONNECT</h4>
+              <img src="/images/social media icons.png">
```

Comment:
- [ ] Please make sure The list of social icons uses a `ul` tag to group them, and `li` tag per icon; each icon has a link (`a` tag).
example: `<ul> <li> <a href='social_link'> <img src=''> </a> </li> </ul>`
*Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.